cfgov/v1/tests/models/test_base.py

```diff
@@ -505,17 +505,17 @@ def test_live_with_another_live_page(self):
         self.check_live_counts(on_live_host=2)
 
 
-class TestCFGOVPageMediaProperty(TestCase):
-    """Tests how the page.media property pulls in child block JS."""
+class TestCFGOVPageMediaJSProperty(TestCase):
+    """Tests how the page.media_js property pulls in child block JS."""
 
     def test_empty_page_has_no_media(self):
-        return self.assertEqual(CFGOVPage().media, [])
+        return self.assertEqual(CFGOVPage().media_js, [])
```

Comment:
These don't need a `return` — the assertion will raise an `AssertionError` if fails, and the test runner catches that exception to fail the test. These look like they predate your change. I'm not sure how many more of these there are in this file? It looks like this is all of them, though.

```suggestion
        self.assertEqual(CFGOVPage().media_js, [])
```